templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -3,12 +3,13 @@
 ![Mention a Team Member](/static/images/help/mention_user.png)
 
 When having a conversation in Zulip, it is sometimes necessary to call
```

Comment:
This doc is switching from third to second person. Try "You may need to call the attention of another user while having a conversation in Zulip"